packages/react/src/firestore/useWaitForPendingWritesQuery.ts

```diff
@@ -0,0 +1,21 @@
+import { useMutation, UseMutationOptions } from "@tanstack/react-query";
+import {
+  Firestore,
+  FirestoreError,
+  waitForPendingWrites,
+} from "firebase/firestore";
+
+type FirestoreUseMutationOptions<TData = unknown, TError = Error> = Omit<
+  UseMutationOptions<TData, TError, void>,
+  "mutationFn"
+>;
+
+export function useWaitForPendingWritesQuery(
+  firestore: Firestore,
+  options?: FirestoreUseMutationOptions<void, FirestoreError>
+) {
+  return useMutation<void, FirestoreError, void>({
```

Comment:
This one is called `useWaitForPendingWritesQuery` but uses `useMutation`

Reply:
my bad at that. fixing it